src/app/map/map.component.ts

```diff
@@ -1,144 +1,145 @@
-//   lng: position.coords.longitude,
-// };
-
-// this.mapCenter = new google.maps.LatLng(point);
\ No newline at end of file
+ import { Component, OnInit, ViewChild } from '@angular/core';
+ import { FormGroup } from '@angular/forms';
+ import { GoogleMap } from '@angular/google-maps';
+ import { Subscription } from 'rxjs';
+ import { Preferences } from '../interfaces/preferences.interface';
+ import { SearchResponse } from '../interfaces/search.response.interface';
+ import { PreferencesFormComponent } from '../preferences-form/preferences-form.component';
+ import { PrefFormService } from '../services/pref-form.service';
+
+ @Component({
+   selector: 'app-map',
+   templateUrl: './map.component.html',
+   styleUrls: ['./map.component.scss']
+ })
+ export class MapComponent implements OnInit {
+   @ViewChild(GoogleMap, { static: false }) map: GoogleMap;
+
+   center: google.maps.LatLngLiteral;
+   lat: number;
+   lng: number;
+
+   markerOptions: google.maps.MarkerOptions = {
+     draggable: false,
+     animation: google.maps.Animation.DROP,
+     // icon = ""
+   };
+
+   mapZoom = 8;
+   mapCenter: google.maps.LatLng;
+   mapOptions: google.maps.MapOptions = {
+     mapTypeId: google.maps.MapTypeId.ROADMAP,
+     zoomControl: true,
+     scrollwheel: false,
+     disableDoubleClickZoom: true
+   };
+
+   formChanged: Subscription;
+   radius = 0;
+   metersToMilesMultiplier = 1609.34;
+
+   markers: google.maps.LatLng[];
+
+   constructor(
+     private prefFormService: PrefFormService
+   ) { }
+
+   ngOnInit() {
+
+     // upon landing, center map on user's current location
+     navigator.geolocation.getCurrentPosition((position) => {
+       this.center = {
+         lat: position.coords.latitude,
+         lng: position.coords.longitude,
+       };
+     });
+
+     this.formChanged = this.prefFormService.formChangedSubject.subscribe((formVals: Preferences) => {
+       //was throwing error so I had to make it a string then parse
+       if (formVals.preferredRadius === null) {
+         this.resetMarkers();
+       }
+       this.radius = parseFloat(formVals.preferredRadius * this.metersToMilesMultiplier + "");
+       formVals.openNow;
+       this.getNearbyBar(this.radius, formVals.openNow, this.center);
+     });
+     this.markers = [];
+   }
+
+   ngAfterViewInit() {
+     this.getCurrentLocation();
+   }
+
+   getCurrentLocation() {
+     navigator.geolocation.getCurrentPosition(
+       (position: GeolocationPosition) => {
+
+         const point: google.maps.LatLngLiteral = {
+           lat: position.coords.latitude,
+           lng: position.coords.longitude,
+         };
+
+         this.mapCenter = new google.maps.LatLng(point);
+         this.map.panTo(point);
+       },
+       (error) => {
+         console.log("Could Not Find User Location " + error);
+       },
+       { enableHighAccuracy: true }
+     );
+   }
+
+   getNearbyBar(radius: number, openNow: boolean, center: any) {
+
```

Comment:
be consistent about whitespace - in some functions you leave a new line between the function signature and the first declaration and others there is no new line